library/src/main/java/com/mux/video/upload/api/MuxUpload.kt

```diff
@@ -318,6 +319,15 @@ class MuxUpload private constructor(
       return this
     }
 
+    /**
+     * If enabled, the Upload SDK will try to standardize the input file in order to optimize it for
+     * use with Mux Video
+     */
+    @Suppress("unused")
+    fun standardizeEnabled(enabled: Boolean) {
+      uploadInfo.update(standardizeEnabled = enabled)
```

Comment:
What about `standardizationEnabled`?

Reply:
I think... How about `standardizationRequested`?

This will probably be temporary, as we'll likely need a `data class` for input params before v1.0 goes out.

Reply:
That's a much better spelling!